samples/react-chat/src/App.tsx

```diff
@@ -13,7 +13,7 @@ export const ChatAddIcon = (): JSX.Element => {
   return <ChatAddIconBundle color={iconColor} />;
 };
 
-const ChatListWrapper = memo(({ onSelected }: { onSelected: (e: GraphChat) => void }) => {
+const ChatListWrapper = memo(({ onSelected }: { onSelected: (e: GraphChatThread) => void }) => {
```

Comment:
I wonder if this type can be converted to `GraphChatThread` from `GraphChat` at the consumer level? Just thinking if `GraphChat` is more generic than `GraphChatThread`.

Reply:
ok, looks like we are extending from base GraphChat so I am good here